server/routes/prisoner.ts

```diff
@@ -28,10 +29,13 @@ export default function routes({ auditService, prisonerProfileService }: Service
       operationId: res.locals.appInsightsOperationId,
     })
 
+    const prisonerDpsAlertsUrl = `${config.dpsPrisoner}prisoner/${prisonerId}/alerts/active`
```

Comment:
Could just build this URL in the template like we do for 'full visit history' button to avoid creating here and passing through 🤷‍♂️. This is fine though